### DISCUSSION TITLE
skip folded tab-trees when using ctrl+pgup/down to move between tabs?

### DISCUSSION THREAD

**Discussion by TheOverpassArsonist** [piroor/treestyletab#3643] [Q&A]
Is there a way to configure it so that the ctrl+pgup/down keybindings to move between tabs will skip folded trees? (or is there some TST way that you should use to navigate through the trees via hotkey?)

**Comment by SHHSSH** [comment]
![image](https://github.com/user-attachments/assets/defdabb5-f7da-4e90-bf90-1bb572a02792)
It's at least something to do with this setting in TST, helper add on Lock as collapsed may help, and I'm currently using these hotkeys, some of which are relatively new TST features so I'd suggest fiddling as I am capable of doing what you're asking - 
![image](https://github.com/user-attachments/assets/54455c20-7aa8-4657-b729-5ec1d3b39db1)
